crypto/test_pairs.ts

```diff
@@ -36,18 +36,51 @@ function pair(secret: string) {
 export function testUser(userId: number) {
   return Sr25519.fromSeed(blake2_256.hash(new TextEncoder().encode(`capi-test-user-${userId}`)))
 }
-
-export function testUserFactory(url: string) {
-  return async function users<N extends number>(count: N): Promise<ArrayOfLength<Sr25519, N>> {
-    const response = await fetch(url, {
-      method: "POST",
-      headers: { "Content-Type": "application/json" },
-      body: JSON.stringify({ count }),
-    })
-    if (!response.ok) throw new Error(await response.text())
-    const { index }: { index: number } = await response.json()
-    const userIds: Sr25519[] = []
-    for (let i = index; i < index + count; i++) userIds.push(testUser(i))
-    return userIds as ArrayOfLength<Sr25519, N>
+export namespace testUser {
```

Comment:
Better as non-namespace IMO

Reply:
I do too. However, as a non-namespace, it's exposed directly from the root mod, even though it's only used in the provider-generated code. I suppose we could manually re-export all non-testUserFactory symbols of `crypto/test_pairs.ts` from `crypto/mod.ts`... but that's less clean.